index.html

```diff
@@ -0,0 +1,227 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;300&family=Roboto:wght@100&display=swap"
+      rel="stylesheet">
+
+  </head>
+
+  <body>
+    <div class="container">
+      <header>
+        <nav>
+          <a href="#" id="welcome">Welcome</a>
+          <a href="#"><img src="images/Union.svg" alt=""></a>
+        </nav>
+      </header>
+      <section class="content">
+        <div class="introduction">
+          <h1 id="name">Hey there. I'm Denis</h1>
+          <h1 id="profession">I'm a software developer</h1>
+        </div>
+        <div class="prelude">
+          <p>
+            I can help you build a product, feature or website. Look through some of my work and experience! If you
+            like what you see and have a project you need coded, don’t hestiate to contact me.
+          </p>
+        </div>
+      </section>
+      <ul class="social">
+        <li><a href="#"><img src="images/twitter.svg" alt=""></a></li>
+        <li><a href="#"><img src="images/linkedin.jpg" alt=""></a></li>
+        <li><a href="#"><img src="images/medium.svg" alt=""></a></li>
+        <li><a href="#"><img src="images/github.svg" alt=""></a></li>
+        <li><a href="#"><img src="images/angellist.svg" alt=""></a></li>
+      </ul>
+    </div>
+
+    <div class="mobile section">
+      <div id="works">
+        <h3>My Recent <br> Works</h3>
+      </div>
+      <div class="img">
+        <img src="images/Indicator.svg" alt="">
+      </div>
```

Comment:
- [  ] You did an amazing job but as previous reviewer suggested wrap work section with tag section using semantic HTML. Kindly change that 💪

Reply:
@dd-obua this is still missing, as suggested by previous reviewers kindly wrap the work section using the `<section><section>` tag instead of div, this makes your website more semantic and easier to read by you and screen browsers